test/disability-benefits/reducers/claims-list.unit.spec.jsx

```diff
@@ -8,19 +7,27 @@ describe('Claims list reducer', () => {
   it('should sort and populate the claims list', () => {
     const claims = Array(12).fill({
       attributes: {
-        updatedAt: '2010-01-04'
+        phaseChangeDate: '2010-01-01'
       }
     });
-    claims[11].attributes.updatedAt = '2011-01-05';
+    claims[11] = {
```

Comment:
Can we add some more data to this test that mimics a scenario where some claims have dates and others only have ID?